### PR TITLE
[MPOM-358] Activate "reporting" profile during releases

### PR DESCRIPTION
**Pull request by kwin**
Only activate m-project-info-reports-p reports during "reporting" profile

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1080,6 +1080,7 @@ under the License.
           <artifactId>maven-release-plugin</artifactId>
           <configuration>
             <autoVersionSubmodules>true</autoVersionSubmodules>
+            <releaseProfiles>apache-release,reporting</releaseProfiles>
```
Comment (slawekjaranowski) [inline pom.xml:1083]: site is not build during release - so this has no effect here

Comment (kwin) [reply pom.xml:1083]: why not? Don't we rely on the default from https://maven.apache.org/maven-release/maven-release-plugin/perform-mojo.html#goals

Comment (slawekjaranowski) [reply pom.xml:1083]: we use only `deploy` goal 
https://github.com/apache/maven-apache-parent/blob/master/pom.xml#L248-L258

Comment (kwin) [reply pom.xml:1083]: Any idea why? IMHO building and deploying site also is very useful (at least for Msven Plugins)

Comment (kwin) [reply pom.xml:1083]: I created https://issues.apache.org/jira/browse/MPOM-359 for that.

Comment (slawekjaranowski) [reply pom.xml:1083]: By the way I am for removing reporting  profile at all - all site reports should be build  always without special profile.
But should be discussed on dev list - maybe exist a rationale reason why we have special profile.

**Comment by jira-importer** [comment]
Resolve #279

**Pull request closed without merging**